cmd/kubeadm/app/constants/constants.go

```diff
@@ -47,6 +47,9 @@ const (
 	// CertificateValidity defines the validity for all the signed certificates generated by kubeadm
 	CertificateValidity = time.Hour * 24 * 365
 
+	// DefaultCertificateDir defines default certificate directory
+	DefaultCertificateDir = "pki"
```

Comment:
depending on https://github.com/kubernetes/kubernetes/pull/103027/files#r655531613 maybe we can avoid this constant.